test/plots/piePlotTests.ts

```diff
@@ -185,7 +233,7 @@ describe("Plots", () => {
         svg.remove();
```

Comment:
On line 199, rename to `data`. There is no `data2` so the name does not make sense.